### ISSUE TITLE
Occassionally failing skip test

### ISSUE THREAD

**Issue by PetrDlouhy** [django-denorm/django-denorm#58]
The "test_app.tests.TestSkip" test sometimes fails with following exception:

```
django.contrib.contenttypes.models.DoesNotExist: ContentType matching query does not exist.
```

See https://travis-ci.org/django-denorm/django-denorm/jobs/88605870 for details.

This bug is very hard to trace, what can I say from rather statistical gues than exact analysis:
-it occurs only on Django 1.7-1.9
-I haven't seen fail on Python 2.7 on my computer, but I have seen fails on Travis
-always fails the "test_with_skip", not "test_meta_skip"
-if I run the "test_with_skip" only, I never saw it fail
-it fails only on SQLite database backend

I would guess that it is some concurrency collision.

**Comment by PetrDlouhy** [comment]
The problem is, that `SkipCommentWithAttributeSkip` is deleted after `test_meta_skip` run. That hits the delete trigger and adds `dirty instance` record. Then the database is cleaned, but that record remains there pointing to nonexistent `content type`.
Then runs the `test_with_skip` test, which bumps into the corrupted record.

Why it happens only sometimes and why the `dirty instance` record arise in the first place is still a mystery.

**Comment by prokaktus** [comment]
Also, all PRs currently failing in Travis due `Conflicting migrations detected (0003_auto_20160119_0522, 0003_auto_20160218_1249 in test_app).`

**Comment by PetrDlouhy** [comment]
It is fixed in PR #66, but unfortunately long time without merging.
